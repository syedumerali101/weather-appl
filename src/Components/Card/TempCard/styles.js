import {StyleSheet} from 'react-native';
import {colors} from '../../../Utils/appTheme';
import {vh} from '../../../Utils/dimensions';

const styles = StyleSheet.create({
  titleText: {
    marginTop: vh * 2,
    color: colors.appBlue,
    fontSize: vh * 2.3,
  },
  descText: {
    color: colors.blackText,
    fontSize: vh * 1.8,
    marginTop: vh * 0.5,
  },
});
export default styles;
