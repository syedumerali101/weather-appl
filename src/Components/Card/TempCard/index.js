import React from 'react';
import {View, Text} from 'react-native';
import styles from './styles';

const TempCard = props => {
  return (
    <View>
      <Text style={styles.titleText}>{props.heading}</Text>
      <Text style={styles.descText}>{props.description}</Text>
      <Text style={styles.descText}>{props.description2}</Text>
    </View>
  );
};
export default TempCard;
