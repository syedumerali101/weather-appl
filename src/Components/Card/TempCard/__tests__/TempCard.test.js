import React from 'react';
import renderer from 'react-test-renderer';
import TempCard from '../index';

test('temp card correctly', () => {
  const tree = renderer.create(<TempCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
