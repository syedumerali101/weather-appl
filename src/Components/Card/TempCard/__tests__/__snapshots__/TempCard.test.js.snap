// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`temp card correctly 1`] = `
<View>
  <Text
    style={
      {
        "color": "#2D73E4",
        "fontSize": 30.682,
        "marginTop": 26.68,
      }
    }
  />
  <Text
    style={
      {
        "color": "#171819",
        "fontSize": 24.012,
        "marginTop": 6.67,
      }
    }
  />
  <Text
    style={
      {
        "color": "#171819",
        "fontSize": 24.012,
        "marginTop": 6.67,
      }
    }
  />
</View>
`;
